website_field_autocomplete/static/src/js/field_autocomplete.js

```diff
@@ -0,0 +1,57 @@
+/* Copyright 2016 LasLabs Inc.
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+ */
+
+odoo.define('website_field_autocomplete.field_autocomplete', function(require){
+  "use strict";
+
+  var snippet_animation = require('web_editor.snippets.animation');
+  var Model = require('web.Model');
+
+  snippet_animation.registry.field_autocomplete = snippet_animation.Class.extend({
+
+    selector: '.s_website_autocomplete',
+
+    start: function() {
+
+      let self = this;
+
+      this.$target.autocomplete({
+        source: function(request, response) {
+          // Must use var, let yields undefined
+          var QueryModel = new Model(self.$target.data('model'));
+          let queryField = self.$target.data('queryField');
```

Comment:
Well that saddens me. I was reading something that it was recommended for all new JS to utilize `let` instead, but this must have been strictly Node. Sometimes I wish there was a better delineation, but I guess that's the point 😛 

Also thanks for the website, this is much better than the one I was using.